src/main/java/com/example/demo/repository/TeamsList.java

```diff
@@ -0,0 +1,61 @@
+package com.example.demo.repository;
+
+import com.example.demo.entity.Team;
+import com.example.demo.service.SQLMethods;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.springframework.stereotype.Repository;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Repository
+@FieldDefaults(level = AccessLevel.PRIVATE)
+public class TeamsList {
+    List<Team> teams = new ArrayList<>();
+    SQLMethods sqlMethods;
+
+    Team Monday = new Team("Monday", "M", "N");
+    Team Tuesday = new Team("Tuesday", "T", "E");
+    Team Wednesday = new Team("Wednesday", "W", "D");
+    Team Thursday = new Team("Thursday", "H", "R");
+    Team Friday = new Team("Friday", "F", "I");
+    Team Saturday = new Team("Saturday", "S", "R");
+    Team Sunday = new Team("Sunday", "D", "A");
+
+    public void madeList() {
```

Comment:
Method name should reflect intention.
Is your intention to create a list in the past?)